Sources/SwiftTwitch/Classes/Results/Users/UserData.swift

```diff
@@ -70,21 +70,4 @@ public struct UserData: Unmarshaling {
     /// permissions for the token provided.
     public let email: String?
 
-    /// Initializes a `UserData` object from the input `MarshaledObject`. This will throw if there
-    /// is missing data from the input `MarshaledObject`.
-    ///
-    /// - Parameter object: The object to initialize a `UserData` object from
-    /// - Throws: If data is missing that was expected to be non-`nil`.
-    public init(object: MarshaledObject) throws {
-        userId = try object.value(for: Twitch.WebRequestKeys.id)
-        userLoginName = try object.value(for: Twitch.WebRequestKeys.login)
```

Comment:
Different property name vs web request key name, need to change variable name or do custom Codable initialization